cache/sampleStoreInit.js

```diff
@@ -41,15 +64,20 @@ function getPreviousStatus() {
 } // persistInProgress
 
 /**
- * Clear all the "sampleStore" keys (for subjects, aspects, samples) from
- * redis.
+ * Clear all the "sampleStore" keys (for subjects, aspects, samples, subaspmap)
+ * from redis.
  *
  * @returns {Promise} upon completion.
  */
 function eradicate() {
   const promises = Object.getOwnPropertyNames(constants.indexKey)
     .map((s) => redisClient.smembersAsync(constants.indexKey[s])
     .then((keys) => {
+      if (constants.indexKey[s] === constants.indexKey.sample) {
+        // this is done to delete keys prefixed with "samsto:subaspmap:"
+        keys.push(...getSubAspMapKeys(keys));
+      }
```

Comment:
this is needed to because we do not have a master list of "subaspmap".  Maybe we can start tracking the "subaspmap" keys in a master list too? 

I think we can just get away by not tracking the mapping of the subjectAbsolutePath to the aspects.  

Any thoughts?